addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderServerBridgeHandler.java

```diff
@@ -1104,26 +1104,22 @@ public void onBridgeDisconnected(ZoneMinderServerBridgeHandler bridge) {
      * Method to start a data refresh task.
      */
     protected ScheduledFuture<?> startTask(Runnable command, long delay, long interval, TimeUnit unit) {
-        ScheduledFuture<?> task = null;
         logger.debug("BRIDGE [{}]: Starting ZoneMinder Bridge Monitor Task. Command='{}'", getThingId(),
                 command.toString());
         if (interval == 0) {
-            return task;
+            return null;
         }
 
-        if (task == null || task.isCancelled()) {
```

Comment:
I have nu clue what happened here over time, but the task was always null in the current code.